internal/ate/ixate.go

```diff
@@ -1379,3 +1381,51 @@ func validateIP(ipc *opb.IpConfig, desc string) error {
 	}
 	return nil
 }
+
+// sends notification/error messages from bgp peers on ATE.
+func (ix *ixATE) SendBGPPeerNotification(ctx context.Context, code int, subCode int, peerNames []string) error {
+
+	if len(peerNames) == 0 {
+		return fmt.Errorf("no bgp peer provided")
+	}
+
+	var cfgNodes []ixconfig.IxiaCfgNode
+	for _, intf := range ix.intfs {
+		for _, bgpName := range peerNames {
+			if val, ok := intf.bgpToBgpIpv4Peer[bgpName]; ok {
+				cfgNodes = append(cfgNodes, val)
+			}
+			if val, ok := intf.bgpToBgpIpv6Peer[bgpName]; ok {
+				cfgNodes = append(cfgNodes, val)
+			}
+		}
+	}
+	if err := ix.c.UpdateIDs(ctx, ix.cfg, cfgNodes...); err != nil {
+		return fmt.Errorf("could not update IDs for bgp peer %w", err)
```

Comment:
per Go style, add a colon to the end of the last word preceding "%w" in all of these returned errors, e.g. "bgp peer: %w" here